clouddriver-google/src/test/groovy/com/netflix/spinnaker/clouddriver/google/security/GoogleNamedAccountCredentialsSpec.groovy

```diff
@@ -49,6 +50,7 @@ class GoogleNamedAccountCredentialsSpec extends Specification implements TestDef
       map.region2 == ['z4', 'z5', 'z6']
   }
 
+  @Ignore("ObjectMapper is failing to convert INSTANCE_TYPE_LIST for some reason?")
```

Comment:
It's not the ObjectMapper failing. The map the ObjectMapper is trying to convert is malformed. Each `machineTypes` key ends up mapping to `null` and the ObjectMapper then faithfully converts the map to a `MachineTypeAggregatedList`.

Remove the `static` from this line and it will pass: https://github.com/spinnaker/clouddriver/blob/master/clouddriver-google/src/test/groovy/com/netflix/spinnaker/clouddriver/google/security/TestDefaults.groovy#L571

Not sure why it worked before or what changed, but seems like we shouldn't have been able to refer to the other static member defined on the trait in the first place.